extra-lisp/ecl/autobuild/defines

```diff
@@ -8,8 +8,7 @@ NOPARALLEL=1
 AUTOTOOLS_AFTER="--enable-unicode=yes \
                  --enable-c99complex \
                  --enable-threads=yes \
-                 --with-__thread \
-                 --disable-rpath \
+                 --enable-rpath=no \
```

Comment:
That shouldn't have made a difference.

Reply:
- with-__theard: deprecated.
- enable-rpath=no: The change is meant to keep with upstream's expression.

So yes, not make a difference.